.github/dependabot.yml

```diff
@@ -17,17 +17,13 @@ updates:
     schedule:
       interval: "weekly"
     ignore:
+      # Managed by cisagov/skeleton-generic
       - dependency-name: actions/cache
       - dependency-name: actions/checkout
+      - dependency-name: actions/setup-go
       - dependency-name: actions/setup-python
-      # Managed by cisagov/skeleton-docker
-      # - dependency-name: actions/download-artifact
-      # - dependency-name: actions/github-script
-      # - dependency-name: actions/upload-artifact
-      # - dependency-name: docker/build-push-action
-      # - dependency-name: docker/login-action
-      # - dependency-name: docker/setup-buildx-action
-      # - dependency-name: docker/setup-qemu-action
```

Comment:
We don't want to delete these- we still want them to trickle downstream where they can be uncommented.

Reply:
Right, I'm not sure what happened there.  I will fix it.